### PR TITLE
templates: add checklist for rebasing to a new version of Fedora

### PR DESCRIPTION
**Pull request by bgilbert**
Based on the [fedora-coreos-config README](https://github.com/coreos/fedora-coreos-config/blob/da55a2e50c2d/README.md#moving-to-a-new-major-version-n-of-fedora).

### CODE REVIEW

**Review by lucab — APPROVED** [review]

**Comment by bgilbert** [comment]
@dustymabe @jlebon PTAL and verify that I got the sequencing right.

**Comment by dustymabe** [comment]
@bgilbert mostly LGTM. I've got some small things to add in a follow up. Should we just link to this template from the fedora-coreos-config README? It could get confusing if there is more than one place for things to live.

**Comment by bgilbert** [comment]
@dustymabe I put the link in the fedora-coreos-config README (https://github.com/coreos/fedora-coreos-config/pull/987) because I couldn't think of a better place to document it.  I'm open to ideas.

**Comment by dustymabe** [comment]
> @dustymabe I put the link in the fedora-coreos-config README ([coreos/fedora-coreos-config#987](https://github.com/coreos/fedora-coreos-config/pull/987)) because I couldn't think of a better place to document it. I'm open to ideas.

Oh man.. You did exactly what I was requesting.. I just didn't see you had already done it! Ignore me :)

File: .github/ISSUE_TEMPLATE/rebase.md [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+
+- [ ] Bump `releasever` in `manifest.yaml`
+- [ ] Update the repos in `manifest.yaml` if needed
+- [ ] Run `cosa fetch --update-lockfile`
+- [ ] PR the result
+
+## Ship rebased `next`
+
+- [ ] Ship `next`
+- [ ] Set a new update barrier for N-2 on `next`. In the barrier entry set a link to [the docs](https://docs.fedoraproject.org/en-US/fedora-coreos/update-barrier-signing-keys/). See [discussion](https://github.com/coreos/fedora-coreos-tracker/issues/480#issuecomment-631724629).
```
[the remark this answers is not in the document]
Comment (jlebon) [reply .github/ISSUE_TEMPLATE/rebase.md:46]: https://github.com/coreos/fedora-coreos-tracker/pull/809

[the remark this answers is not in the document]
Comment (bgilbert) [reply .github/ISSUE_TEMPLATE/rebase.md:46]: (or https://github.com/coreos/fedora-coreos-tracker/pull/810, mid-air collision :boom:)

**Pull request merged**